src/test/testdata/expected/lsl/literalArrays.lsl

```diff
@@ -1,20 +1,29 @@
 libsl "1.0.0";
 library literalArrays;
 typealias Int = int32;
-typealias ArrayType = array<Int>;
+typealias IntArray = array<Int>;
+typealias String = string;
+typealias StringArray = array<String>;
 define action TEST_ACTION(
-    a: ArrayType
+    a: IntArray
+);
+define action TEST_ACTION(
+    s: StringArray
+);
+define action TEST_ACTION_ANY(
+    any: any
```

Comment:
"any" is keyword or not ? If this is keyword - can we give to arguments names which equal to keyword ?